### PR TITLE
Pass additional parameters through to AppCfg.

### PR DESCRIPTION
**Pull request by josephburnett**
Re: https://github.com/GoogleCloudPlatform/appengine-maven-plugin/issues/67

### CODE REVIEW

File: src/main/java/com/google/appengine/appcfg/AbstractAppCfgMojo.java [hunk 1]
```diff
@@ -240,6 +240,13 @@
    */
   protected String instance;
 
+  /**
+   * Additional parameters to pass through to appcfg.py.
+   *
```
Comment (ludoch) [inline src/main/java/com/google/appengine/appcfg/AbstractAppCfgMojo.java]: Thanks for doing that. This is for appcfg (.java edition) not the .py edition.

File: src/main/java/com/google/appengine/appcfg/AbstractAppCfgMojo.java [hunk 2]
```diff
@@ -370,6 +377,10 @@ protected void executeAppCfgBackendsCommand(String action, String appDir)
       arguments.add("--enable_jar_classes");
     }
 
+    if (additionalParams != null && !additionalParams.isEmpty()) {
+      arguments.add(additionalParams);
```
Comment (ludoch) [inline src/main/java/com/google/appengine/appcfg/AbstractAppCfgMojo.java]: Not sure about the behavior when you want more than 1 flag and the space separator handled by the Java Process build list of arguments. I think to be 100% portable (Windows, Linux) we might have to do an arguments.add for each flag (black separated) of the additionalParams.

File: src/main/java/com/google/appengine/appcfg/AbstractAppCfgMojo.java (hunk 1 above)
Comment (josephburnett) [inline src/main/java/com/google/appengine/appcfg/AbstractAppCfgMojo.java]: Corrected.

File: src/main/java/com/google/appengine/appcfg/AbstractAppCfgMojo.java (hunk 2 above)
Comment (josephburnett) [inline src/main/java/com/google/appengine/appcfg/AbstractAppCfgMojo.java]: Changed parameter to String[].

**Comment by ludoch** [comment]
LGTM

**Pull request merged**